test_helpers/integration_helpers_test.go

```diff
@@ -48,7 +48,7 @@ var _ = Describe("helper functions for integration tests", func() {
 								"director_uuid": "some director UUID"
```

Comment:
You must have new tests for these changes. I can think of at least two test to parse/marshall the new VM properties. Also, you need to have tests for the defaults and need test for the fact that `create_vm` has now changed.

Finally, you need to add a top-level integration test (like @posijon's tests for has_vm) for this.